deps/chakrashim/tools/update-changelog.js

```diff
@@ -179,6 +185,15 @@ function pipe(proc, onLine) {
   return proc;
 }
 
+function command_sync(command, args) {
+  var x = child_process.spawnSync(command, args);
+  return x.status == 0 ? x.stdout.toString(): x.stderr.toString();
+}
+
+function git_sync(args) {
+  return command_sync('git', args);
```

Comment:
one common issue with gitsync operation is that multiple gitsync op are not allowed at a time. Is it possible that another process might operate under the same local repo?